### ISSUE TITLE
Custom icons in the Console

### ISSUE THREAD

**Issue by burningice2866** [Orckestra/C1-CMS-Foundation#436]
In the old UI it was possible to use own icons by using a different namespace than the default [*Composite.Icons*](https://github.com/Orckestra/C1-CMS-Foundation/blob/dev/Composite/Core/ResourceSystem/Icons/BuildInIconProviderName.cs#L10) when creating a [ResourceHandle](https://github.com/Orckestra/C1-CMS-Foundation/blob/dev/Composite/Core/ResourceSystem/ResourceHandle.cs#L15),, but this setting doesn't seem to be honored any longer. All icons seems to be taken from your own sprite.svg no matter what value i use as the ResourceNamespace.

So my question is, how do i instruct the console to reference a particular icon from a different spritesheet than your own by passing my own ResourceNamespace when instantiating a ResourceHandle.

**Comment by mawtex** [comment]
True, icons cannot be aggregated from different sources as things stand now - you need to add SVGs to the Website\Composite\images\icons\svg folder and run a grunt task to include new icons, which lead to maintenance issues.

We are very much open to adding icons to the core - as long as they have some broader relevance and licensing allow for it.

I'll mark this as an enhancement.

**Comment by burningice2866** [comment]
I'm not interested in adding the icons in question to the core since they would be quite specific to a single package of mine. 

You should mark the ResourceNamespace as obsolotete then though, and add a constructor overload that only takes a resourceName argument.
